src/goto-programs/abstract-interpretation/interval_domain.cpp

```diff
@@ -1219,6 +1235,13 @@ void interval_domaint::assume_rec(const expr2tc &cond, bool negation)
     else if(is_floatbv_type(cond) && enable_real_intervals)
       apply_assume_symbol_truth<real_intervalt>(to_symbol2t(cond), negation);
   }
+  else if(is_constant_bool2t(cond))
```

Comment:
Why do we need this? Add some explanation.

Reply:
Done.
Added "// //added in case "cond = false" which happens when the ibex contractor results in empty set. "